website/docs/docs/dbt-cloud-environments.md

```diff
@@ -15,8 +15,9 @@ Critically, in order to execute dbt, environments define three variables:
 
 Each dbt Cloud project can have only one [development environment](#create-a-development-environment), but there is no limit to the number of [deployment environments](/docs/deploy/deploy-environments), providing you the flexibility and customization to tailor the execution of scheduled jobs. 
 
-Use environments to customize settings for different stages of your project and streamline the execution process by using software engineering principles. This page will detail the different types of environments and how to intuitively configure your development environment in dbt Cloud. 
+<Lightbox src="/img/dbt-env.png" width="90%" title="dbt Cloud environment hierarchy showing projects, environments, connections, and orchestration jobs." />
 
+Use environments to customize settings for different stages of your project and streamline the execution process by using software engineering principles. This page will detail the different types of environments and how to intuitively configure your development environment in dbt Cloud. 
```

Comment:
I wonder if this 1st sentence can move above the image to help describe what the image is conveying.
I mistook "this page" to mean the image at first. Maybe rewording might help.

```suggestion
The following sections detail the different types of environments and how to intuitively configure your development environment in dbt Cloud. 
```